libkpod/container.go

```diff
@@ -5,73 +5,74 @@ import (
 	"github.com/pkg/errors"
 )
 
-// FindContainer searches for a container with the given name or ID in the given store
-func FindContainer(store cstorage.Store, container string) (*cstorage.Container, error) {
-	return store.Container(container)
+// GetStorageContainer searches for a container with the given name or ID in the given store
+func (c *ContainerServer) GetStorageContainer(container string) (*cstorage.Container, error) {
+	// TODO: make this check ctr indexes first
```

Comment:
If this TODO needs to be taken care of before this can be merged, then this will have to wait until #717 is merged